src/components/EndorsementChain/EndorsementChainLayout/EndorsementChainLayout.tsx

```diff
@@ -125,15 +124,25 @@ export const EndorsementChainLayout: FunctionComponent<EndorsementChainLayout> =
         // ui for each surrender row
         case "Surrender":
           tableRows.push(
-            <SimpleTableRow
-              key={tableRowIndex++}
-              index={tableRowIndex++}
-              date={format(new Date(tradetrustErc721Event?.eventTimestamp ?? 0), "do MMM yyyy, hh:mm aa")}
-              actionTitle="Document surrendered to issuer"
-              displayDashHead={true}
-              displayDot={true}
-              displayDashTail={!isLastEvent}
-            />
+            <tr className="table-row" key={tableRowIndex++}>
+              <td className="table-cell">
+                <div className="action-title">Document surrendered to issuer</div>
+                <div className="date">
+                  {format(new Date(tradetrustErc721Event?.eventTimestamp ?? 0), "do MMM yyyy, hh:mm aa")}
+                </div>
+              </td>
+              <td className="table-cell">
+                <AddressCell address="" isNewAddress={true} displayDashHead={true} displayDashTail={!isLastEvent} />
+              </td>
+              <td className="table-cell">
+                <AddressCell
+                  address=""
+                  isNewAddress={false}
+                  displayDashHead={!isLastEvent && noOfTransferActions !== totalNumberOfTransferActions}
+                  displayDashTail={!isLastEvent && noOfTransferActions !== totalNumberOfTransferActions}
+                />
+              </td>
+            </tr>
           );
           break;
 
```

Comment:
unrelated but since u're at it, able to move the styles to div if possible? then remove `EndorsementChainLayoutStyle` entirely